diff -Nur man-1.5i2.orig/src/apropos.sh man-1.5i2/src/apropos.sh
--- man-1.5i2.orig/src/apropos.sh	Mon Mar 19 17:14:56 2001
+++ man-1.5i2/src/apropos.sh	Sun Oct 14 11:01:36 2001
@@ -67,7 +67,7 @@
 found=0
 while [ $found = 0 -a -n "$1" ]
 do
-    for d in $manpath /usr/lib
+    for d in /var/cache/man /usr/lib
     do
         if [ -f $d/whatis ]
         then
@@ -101,7 +101,7 @@
     done
     nothing=
     found=0
-    for d in $manpath /usr/lib
+    for d in /var/cache/man /usr/lib
     do
         if [ -f $d/whatis ]
         then
diff -Nur man-1.5i2.orig/src/makewhatis.sh man-1.5i2/src/makewhatis.sh
--- man-1.5i2.orig/src/makewhatis.sh	Sun Oct 14 11:01:23 2001
+++ man-1.5i2/src/makewhatis.sh	Sun Oct 14 11:03:00 2001
@@ -122,7 +122,7 @@
       eval path="\$$pages"path
       for mandir in $path
       do
-	 cp /dev/null $mandir/whatis
+	 cp /dev/null /var/cache/man/whatis
       done
    done
 fi
@@ -136,7 +136,7 @@
      if [ x$verbose != x ]; then
 	echo "about to enter $mandir" > /dev/tty
      fi
-     if [ -s ${mandir}/whatis -a $pages = man -a x$update = x ]; then
+     if [ -s /var/cache/man/whatis -a $pages = man -a x$update = x ]; then
 	if [ x$verbose != x ]; then
 	   echo skipping $mandir - we did it already > /dev/tty
 	fi
@@ -350,13 +350,13 @@
 	 mandir1=$mandir
        fi
 
-       if [ -f ${mandir1}/whatis ]
+       if [ -f /var/cache/man/whatis ]
        then
-	 cat ${mandir1}/whatis >> $TMPFILE
+	 cat /var/cache/man/whatis >> $TMPFILE
        fi
-       sed '/^$/d' < $TMPFILE | sort | uniq > ${mandir1}/whatis
+       sed '/^$/d' < $TMPFILE | sort | uniq > /var/cache/man/whatis
 
-       chmod 644 ${mandir1}/whatis
+       chmod 644 /var/cache/man/whatis
        rm $TMPFILE
      fi
    done
