--- man-1.5l/configure.orig	Tue Mar 11 21:54:36 2003
+++ man-1.5l/configure	Tue Mar 11 21:56:38 2003
@@ -473,20 +473,20 @@
   then
     if test $Fnroff = "missing"
     then
-      nroff="nroff -Tlatin1 -mandoc"
+      nroff="nroff -msafer -Tlatin1 -mandoc"
     else
-      nroff="$Fnroff -Tlatin1 -mandoc"
+      nroff="$Fnroff -msafer -Tlatin1 -mandoc"
     fi
-    troff="troff -mandoc"
+    troff="troff -msafer -mandoc"
     echo "Warning: could not find groff"
   else
     if test $Fnroff = "missing"
     then
-      nroff="$Fgroff -Tlatin1 -mandoc"
+      nroff="$Fgroff -S -Tlatin1 -mandoc"
     else
-      nroff="$Fnroff -Tlatin1 -mandoc"
+      nroff="$Fnroff -msafer -Tlatin1 -mandoc"
     fi
-    troff="$Fgroff -Tps -mandoc"
+    troff="$Fgroff -S -Tps -mandoc"
     jnroff="$Fgroff -Tnippon -mandocj"
   fi
   eqn="$Fgeqn -Tps"
