--- man-1.5p/src/gripes.c	2005-01-08 08:01:21.000000000 +0000
+++ man-1.5p.new/src/gripes.c	2005-03-07 21:55:54.522402480 +0000
@@ -21,7 +21,20 @@
 	fprintf (stderr, "man: internal error - cannot find message %d\n", n);
 	exit (1);
     }
-    return s;
+    /* routine to filter format string abuse.  will */
+    /* only allow %d, %s, and %o through.  no more  */
+    /* than two formats needed for any response.    */
+    for (i = 0; s[i] != 0x0; i++){
+        if (s[i] == '%' && s[i+1]){
+            if (strchr("dso", s[i+1])) /* %d,%s,%o. */
+                fmt_n++;
+            else
+                fmt_n=3; /* anything else = <limit. */
+        }
+       if (fmt_n > 2) /* failed, default reply. */
+            s = msg[n];
+    }
+	return s;
 }
 
 #else /* NONLS */
@@ -99,6 +112,8 @@
 static char *
 getmsg (int n) {
 	char *s = "";
+    unsigned int i = 0;
+    unsigned short fmt_n = 0;
 
 	catinit ();
 	if (catfd != (nl_catd) -1) {
