diff -Naur man-1.5k/src/makewhatis.sh man-1.5k-p/src/makewhatis.sh
--- man-1.5k/src/makewhatis.sh	Tue Jun 29 04:20:59 1999
+++ man-1.5k-p/src/makewhatis.sh	Fri Jul 14 17:39:43 2000
@@ -36,8 +36,8 @@
 # In case /usr is read-only, make /usr/man/whatis (etc) a symlink to
 # something like /var/cache/man/whatis.
 
-# AWK=/usr/bin/gawk
-AWK=%awk%
+AWK=/bin/awk
+# AWK=%awk%
 
 # Find a place for our temporary files. If security is not a concern, use
 #	TMPFILE=/tmp/whatis$$; TMPFILEDIR=none
