diff -Nur man-1.5k.orig/catopen/catopen.c man-1.5k/catopen/catopen.c
--- man-1.5k.orig/catopen/catopen.c	Sun Oct 14 11:05:49 2001
+++ man-1.5k/catopen/catopen.c	Sun Oct 14 11:06:00 2001
@@ -27,14 +27,6 @@
     /* The quotes below are from X/Open, XPG 1987, Vol. 3. */
 
     /*
-     * "In catopen(), the oflag argument is reserved for future use
-     * and should be set to 0."
-     */
-
-    if (oflag)
-      return fd;		/* only treat the known case */
-
-    /*
      * "If `name' contains a `/', then `name' specifies a pathname"
      */
     if (index(name, '/')) {
@@ -87,8 +79,12 @@
 
   namesz = strlen(name);
 
-  lang = getenv("LANG");
+  lang = getenv("LC_ALL");
+  if (!lang)
+    lang = getenv("LC_MESSAGES");
   if (!lang)
+    lang = getenv("LANG");
+  if (!lang || !strcmp(lang,"C") || !strcmp(lang,"POSIX"))
     lang = "";
   langsz = strlen(lang);
 
diff -Nur man-1.5k.orig/src/gripes.c man-1.5k/src/gripes.c
--- man-1.5k.orig/src/gripes.c	Mon Mar 19 21:38:00 2001
+++ man-1.5k/src/gripes.c	Sun Oct 14 11:06:00 2001
@@ -35,18 +35,19 @@
 static void
 catinit (void) {
     if (!cat_is_open) {
-	catfd = my_catopen(mantexts,0);
+	catfd = my_catopen(mantexts,NL_CAT_LOCALE);
 	if (catfd == (nl_catd) -1) {
 	    /* Only complain if at least one of NLSPATH and LANG exists,
 	       and LANG != "en" (or when debugging). Also accept en_ZA etc. */
 	    char *s, *lg;
 	    s = getenv("NLSPATH");
-	    lg = getenv("LANG");
+	    lg = getenv("LC_ALL");
 	    if (!lg)
 		    lg = getenv("LC_MESSAGES");
 	    if (!lg)
-		    lg = getenv("LC_ALL");
-	    if ((s || lg) && (!lg || strncmp(lg, "en", 2))) {
+		    lg = getenv("LANG");
+	    if ((s || lg) && (!lg || (strncmp(lg, "en", 2) &&
+			  strncmp(lg, "C", 1) && strncmp(lg, "POSIX", 5)))) {
 		perror(mantexts);
 		fprintf(stderr,
 "Failed to open the message catalog %s on the path NLSPATH=%s\n\n",
diff -Nur man-1.5k.orig/src/man.c man-1.5k/src/man.c
--- man-1.5k.orig/src/man.c	Sun Oct 14 11:05:49 2001
+++ man-1.5k/src/man.c	Sun Oct 14 11:06:00 2001
@@ -1217,11 +1217,7 @@
      CHECK("LC_MESSAGES", 128);
      CHECK("LC_ALL", 128);
 
-#ifndef __FreeBSD__ 
-     /* Slaven Rezif: FreeBSD-2.2-SNAP does not recognize LC_MESSAGES. */
-     setlocale(LC_CTYPE, "");	/* used anywhere? maybe only isdigit()? */
-     setlocale(LC_MESSAGES, "");
-#endif
+     setlocale(LC_ALL, "");
 
      /* No doubt we'll need some generic language code here later.
 	For the moment only Japanese support. */
diff -Nur man-1.5k.orig/src/manpath.c man-1.5k/src/manpath.c
--- man-1.5k.orig/src/manpath.c	Tue Jun  5 23:47:09 2001
+++ man-1.5k/src/manpath.c	Sun Oct 14 11:10:30 2001
@@ -273,13 +273,11 @@
 	} else {
 		/* We cannot use "lang = setlocale(LC_MESSAGES, NULL)" or so:
 		   the return value of setlocale is an opaque string. */
-		if((lang = getenv("LANG")) != NULL)
-			split2(dir, lang, add_to_mandirlist_x, perrs);
-		if((lang = getenv("LANGUAGE")) != NULL)
-			split2(dir, lang, add_to_mandirlist_x, perrs);
-		if((lang = getenv("LC_MESSAGES")) != NULL)
-			split2(dir, lang, add_to_mandirlist_x, perrs);
-		if((lang = getenv("LC_ALL")) != NULL)
+		lang = getenv("LC_ALL");
+		if(!lang) lang=getenv("LC_MESSAGES");
+		if(!lang) lang=getenv("LANG");
+		if(!lang) lang=getenv("LANGUAGE");
+		if(lang != NULL)
 			split2(dir, lang, add_to_mandirlist_x, perrs);
 		add_to_mandirlist_x(dir, 0, perrs);
 	}
