--- man-1.5n/msgs/mess.bg.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.bg.codeset	2004-08-03 22:26:00.271574024 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=cp1251
--- man-1.5n/msgs/mess.cs.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.cs.codeset	2004-08-03 22:26:07.721441472 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=iso-8859-2
--- man-1.5n/msgs/mess.el.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.el.codeset	2004-08-03 22:26:24.343914472 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=iso-8859-7
--- man-1.5n/msgs/mess.hr.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.hr.codeset	2004-08-03 22:27:34.066315056 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=iso-8859-2
--- man-1.5n/msgs/mess.ja.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.ja.codeset	2004-08-03 22:27:44.852675280 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=euc-jp
--- man-1.5n/msgs/mess.ko.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.ko.codeset	2004-08-03 22:28:38.301549816 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=euc-kr
--- man-1.5n/msgs/mess.pl.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.pl.codeset	2004-08-03 22:29:30.000690352 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=iso-8859-2
--- man-1.5n/msgs/mess.ro.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.ro.codeset	2004-08-03 22:29:39.272280856 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=iso-8859-2
--- man-1.5n/msgs/mess.sl.codeset.orig	2004-07-15 15:42:16.000000000 +0200
+++ man-1.5n/msgs/mess.sl.codeset	2004-08-03 22:29:50.754535288 +0200
@@ -1 +1 @@
-$ codeset=iso-8859-1
+$ codeset=iso-8859-2
