diff -ur man-1.5p/man2html/Makefile.in man-1.5p.new/man2html/Makefile.in
--- man-1.5p/man2html/Makefile.in	2005-01-08 00:49:37.000000000 +0000
+++ man-1.5p.new/man2html/Makefile.in	2005-03-07 21:38:26.480729032 +0000
@@ -31,13 +31,13 @@
 	mkdir -p $(httpdir)/cgi-aux/man
 	install -m 755 scripts/cgi-bin/man/* $(httpdir)/cgi-bin/man
 	install -m 644 scripts/cgi-aux/man/* $(httpdir)/cgi-aux/man
-	install -d -o $(cgiowner) -g $(cgigroup) -m 775 $(vardir)/man2html
+	install -d -m 775 $(vardir)/cache/man2html
 # (aux was renamed to cgi-aux since aux causes problems under DOS)
 
 # If you have installed glimpse, and have compressed man pages,
 # then perhaps you also want these filters.
 install-glimpse-stuff:
-	install -m 644 glimpse_filters $(vardir)/man2html/.glimpse_filters
+	install -m 644 glimpse_filters $(vardir)/cache/man2html/.glimpse_filters
 
 # In order not to have to type a long command like
 #   netscape http://localhost/cgi-bin/man/man2html?section+topic
diff -ur man-1.5p/man2html/scripts/cgi-bin/man/mansearch man-1.5p.new/man2html/scripts/cgi-bin/man/mansearch
--- man-1.5p/man2html/scripts/cgi-bin/man/mansearch	2003-03-11 15:53:30.000000000 +0000
+++ man-1.5p.new/man2html/scripts/cgi-bin/man/mansearch	2005-03-07 21:39:11.761845256 +0000
@@ -62,7 +62,7 @@
   sedcmd = ENVIRON["SED"];
   truncate_at = 11;		# Single page display match limit.
 
-  glimpse_cmd = "glimpse -z -H /var/man2html -y -W -i "
+  glimpse_cmd = "glimpse -z -H /var/cache/man2html -y -W -i "
 
   for (i = 1; i < ARGC; i++) {
     string = string " " ARGV[i];
diff -ur man-1.5p/man2html/scripts/cgi-bin/man/mansec man-1.5p.new/man2html/scripts/cgi-bin/man/mansec
--- man-1.5p/man2html/scripts/cgi-bin/man/mansec	2004-08-23 00:17:40.000000000 +0000
+++ man-1.5p.new/man2html/scripts/cgi-bin/man/mansec	2005-03-07 21:39:33.692511288 +0000
@@ -40,7 +40,7 @@
     cache_suffix = "h";
   }
 
-  cache_dir  = "/var/man2html";
+  cache_dir  = "/var/cache/man2html";
   cache_file = "manindex" cache_suffix "-" section ".html";
   cache = cache_dir "/" cache_file;
   cache_tmp = cache "_" pid;
diff -ur man-1.5p/man2html/scripts/cgi-bin/man/manwhatis man-1.5p.new/man2html/scripts/cgi-bin/man/manwhatis
--- man-1.5p/man2html/scripts/cgi-bin/man/manwhatis	2003-03-11 15:53:29.000000000 +0000
+++ man-1.5p.new/man2html/scripts/cgi-bin/man/manwhatis	2005-03-07 21:39:47.450419768 +0000
@@ -39,7 +39,7 @@
     cache_suffix = "h";
   }
 
-  cache_dir  = "/var/man2html";
+  cache_dir  = "/var/cache/man2html";
   cache_file = "whatis" cache_suffix "-" section ".html";
   cache = cache_dir "/" cache_file;
