--- man-1.5l/configure.orig	Mon Mar 10 21:21:00 2003
+++ man-1.5l/configure	Tue Mar 11 21:52:34 2003
@@ -183,7 +183,7 @@
 # All documentation assumes "man.conf" - in case you invent some other name
 # (why?), also man pages and other docs need to be patched.
 if [ -z "${prefix}" ]; then prefix="/usr"; fi
-if [ -z "${confprefix}" ]; then confprefix=${prefix}; fi
+if [ -z "${confprefix}" ]; then confprefix=""; fi
 if [ -z "${bindir}" ]; then bindir="${prefix}/bin"; fi
 if [ -z "${sbindir}" ]; then sbindir="${prefix}/sbin"; fi
 if [ -z "${mandir}" ]; then mandir="${prefix}/man"; fi
@@ -197,7 +197,7 @@
     confdir="${confprefix}/share/misc"
   fi
 fi
-conffilename="man.conf"
+conffilename="man.config"
 
 if test "$RANDOM" = "$RANDOM"; then
   # Plain old Bourne shell.
