diff -ur man.old/pl/apropos.man man/pl/apropos.man
--- man.old/pl/apropos.man	Tue Dec  2 14:23:31 1997
+++ man/pl/apropos.man	Wed Oct 17 00:35:17 2001
@@ -14,9 +14,9 @@
 .\"
 .\" T³umaczenie: 950322 Rafa³ Maszkowski <rzm@pdi.net>
 .\"
-.TH apropos 1 "Jan 15, 1991"
+.TH apropos 1 "15 stycznia 1991"
 .LO 1
-.SH NAME
+.SH NAZWA
 apropos \- szuka ³añcuchów znakow w bazie whatis
 .SH SK£ADNIA
 .BI apropos
@@ -25,5 +25,5 @@
 apropos szuka s³ów kluczowych w plikach bazy danych, które
 zawieraj± krótkie opisy komend systemowych i wy¶wietla wynik
 u¿ywaj±c standardowego wyj¶cia.
-.SH "ZOBACZ TE¯"
+.SH "ZOBACZ TAK¯E"
 whatis(1), man(1).
diff -ur man.old/pl/man.conf.man man/pl/man.conf.man
--- man.old/pl/man.conf.man	Tue Dec  2 18:58:49 1997
+++ man/pl/man.conf.man	Wed Oct 17 00:35:59 2001
@@ -1,7 +1,7 @@
 .\" @(#)man.conf
 .\" T³umaczenie: 950322 Rafa³ Maszkowski <rzm@pdi.net>
-.TH MAN.CONF 5 "30 Mar 1994"
-.SH NAME
+.TH MAN.CONF 5 "30 marca 1994"
+.SH NAZWA
 man.conf \- zawiera konfiguracjê programu man
 .SH OPIS
 .LP
@@ -36,6 +36,6 @@
 .RE
 .LP
 ¿eby wy³±czyæ podkre¶lanie i wytluszczanie.
-.SH "ZOBACZ TE¯"
+.SH "ZOBACZ TAK¯E"
 col(1), (g)eqn(1), (g)pic(1), groff(1), grotty(1), (g)refer(1), (g)tbl(1),
 less(1), man (1) i compress(1), gzip(1).
diff -ur man.old/pl/whatis.man man/pl/whatis.man
--- man.old/pl/whatis.man	Tue Dec  2 14:23:31 1997
+++ man/pl/whatis.man	Wed Oct 17 00:35:37 2001
@@ -14,9 +14,9 @@
 .\"
 .\" T³umaczenie - 950322 Rafa³ Maszkowski <rzm@pdi.net>
 .\"
-.TH whatis 1 "Jan 5, 1991"
+.TH whatis 1 "5 stycznia 1991"
 .LO 1
-.SH NAME
+.SH NAZWA
 whatis \- szukaj s³ów w bazie whatis.
 .SH SK£ADNIA
 .BI whatis
@@ -26,5 +26,5 @@
 opisy komend systemowych i wy¶wietla rezultat przez standardowe
 wyj¶cie. Wy¶wietlane s± tylko komendy, których nazwy pasuj± dok³adnie.
 Baza whatis jest tworzona przy u¿yciu komendy @makewhatis@.
-.SH "ZOBACZ TE¯"
+.SH "ZOBACZ TAK¯E"
 apropos(1), man(1).
diff -ur man.old/pl/man.man man/pl/man.man
--- man.old/pl/man.man	Fri Nov  2 13:20:26 2001
+++ man/pl/man.man	Fri Nov  2 22:56:03 2001
@@ -16,193 +16,364 @@
 .\"
 .\" T³umaczenie - 950322 Rafa³ Maszkowski <rzm@pdi.net>
 .\" 960619 - aktualizacja -K
+.\" 20011102 - aktualizacja do wersji z man-1.5i2, czê¶ciowo w oparciu
+.\"            o alternatywne t³umaczenie PB z PTM - AMK
 .\"
-.TH man 1 "December 20, 1994"
+.TH man 1 "2 wrze¶nia 1995"
 .LO 1
-.SH NAME
-man \- formatowanie i wy¶wietlanie dostêpnych stron manuala
+.SH NAZWA
+man \- formatowanie i wy¶wietlanie dostêpnych stron podrêcznika man
 .br
-manpath \- wyznaczanie ¶cie¿ki poszukiwania stron manuala odpowiedniej dla danego u¿ytkownika
+manpath \- wyznaczanie ¶cie¿ki poszukiwania stron podrêcznika odpowiedniej dla
+danego u¿ytkownika
 .SH SK£ADNIA
-man [\-adfhkKtw] [\-m system] [\-p lañcuch znaków] [\-C plik_konfiguracji] [\-M ¶cie¿ka]
-[\-P pager] [\-S lista_sekcji] [sekcja] nazwa ...
+.TP 4
+.B man
+.RB [ \-adfhkKtwW ]
+.RB [ \-m
+.IR system ]
+.RB [ \-p
+.IR ³añcuch_znaków ]
+.RB [\-C
+.IR plik_konfiguracyjny ]
+.RB [ \-M
+.IR ¶cie¿ka ]
+.RB [ \-P
+.IR pager ]
+.RB [ \-S
+.IR lista_rozdzia³ów ]
+.RI [ rozdzia³ ]
+.IR nazwa ...
 .SH OPIS
 .B man
-formatuje i wy¶wietla strony manuala dostêpne w systemie. Ta wersja
-sprawdza zmienne ¶rodowiskowe
-.B MANPATH
-i
-.BR (MAN)PAGER ,
-wiêc mo¿esz mieæ Twój
-w³asny zestaw (lub zestawy) osobistych stron manuala i wybraæ do
-wy¶wietlania ulubiony program. Je¿eli zostanie podana
-.I sekcja,
-.B man
-sprawdza tylko t± sekcjê manuala.
-U¿ywaj±c opcji lub zmiennych ¶rodowiskowych mo¿esz tak¿e podaæ porz±dek,
-w jakim nale¿y przeszukiwaæ sekcje oraz jakie preprocesory powinny
-zostaæ u¿yte do wstêpnego przetworzenia stron.
+formatuje i wy¶wietla dostêpne w systemie strony podrêcznika man.
+.\" Ta wersja sprawdza zmienne ¶rodowiskowe
+.\" .B MANPATH
+.\" i
+.\" .BR (MAN)PAGER ,
+.\" wiêc mo¿esz mieæ swój w³asny zestaw stron podrêcznika man i wybraæ do ich
+.\" wy¶wietlania ulubiony program.
+Je¿eli zostanie podany
+.IR rozdzia³ ,
+.B man
+sprawdza tylko ten rozdzia³ podrêcznika.
+.\" U¿ywaj±c opcji lub zmiennych ¶rodowiskowych mo¿esz tak¿e podaæ porz±dek,
+.\" w jakim nale¿y przeszukiwaæ rozdzia³y oraz jakie preprocesory powinny
+.\" zostaæ u¿yte do wstêpnego przetworzenia stron.
 Je¿eli
 .I nazwa
-zawiera / wtedy man próbuje najpierw znale¼æ podany plik, wiêc np.
-mo¿esz
+zawiera znak /, to man próbuje najpierw znale¼æ podany plik, wiêc mo¿na np.
+wydaæ polecenie
 .B "man ./foo.5"
 a nawet
 .B "man /cd/foo/bar.1.gz\fR.\fP"
+.PP
+Ni¿ej znajdziesz informacje, gdzie
+.B man
+szuka plików stron podrêcznika.
+
 .SH OPCJE
 .TP
-.B \-\^C " plik_konfiguracji"
-Mo¿esz podaæ inny man.conf; domy¶lnie jest to
-@man_config_file@. (Zobacz man.conf(5).)
+.B \-\^C " plik_konfiguracyjny"
+Okre¶la plik konfiguracyjny, którego nale¿y u¿yæ. Domy¶lnym jest
+.BR @man_config_file@ .
+(Zobacz
+.BR man.conf (5).)
 .TP
 .B \-\^M " ¶cie¿ka"
-Mo¿esz podaæ listê katalogów, w których bêd± szukane strony manuala.
-Je¿eli taka opcja nie zostanie podana, u¿ywana jest zmienna ¶rodowiskowa
-.B MANPATH
-. Je¿eli taka zmienna nie jest zdefiniowana, domy¶lna lista jest
-formowana przez sprawdzenie @man_config_file@. Pusta MANPATH równie¿
-oznacza wybranie ¶cie¿ki domy¶lnej.
+Okre¶la listê katalogów, w których bêd± szukane strony podrêcznika man.
+Katalogi nale¿y rozdzielaæ dwukropkami. Pusta lista jest równoznaczna z nie
+podaniem tej opcji. Zobacz
+.BR "¦CIE¯KA PRZESZUKIWANIA DLA STRON PODRÊCZNIKA" .
 .TP
 .B \-\^P " pager"
-Mo¿esz podaæ jakiego pagera chcesz uzywaæ. Domy¶lnie man u¿ywa
-.B @pager@
-Podanie tej opcji powoduje zignorowanie wartosci zmiennej ¶rodowiskowej
-.B PAGER
-.
-.TP
-.B \-\^S " lista_sekcji"
-Oddzielona dwukropkami lista sekcji manuala, które maja byæ
-przeszukane. Je¿eli uzyta jest ta opcja, zmienna ¶rodowiskowa
+Okre¶la, który pager ma zostaæ u¿yty.  Je¿eli podana jest ta opcja,
+ignorowana jest zmienna ¶rodowiskowa
+.BR MANPAGER ,
+której u¿ycie z kolei powoduje zignorowanie zmiennej ¶rodowiskowej
+.BR PAGER .
+Domy¶lnie
+.B man
+u¿ywa
+.BR @pager@ .
+.TP
+.B \-\^S " lista_rozdzia³ów"
+Rozdzielona dwukropkami lista rozdzia³ów podrêcznika, które maj± byæ
+przeszukane. Je¿eli podana jest ta opcja, zmienna ¶rodowiskowa
 .B MANSECT
 jest ignorowana.
 .TP
 .B \-\^a
-Zwykle man zakoñczy dzia³anie po wy¶wietleniu pierwszej znalezionej
-strony. U¿ycie tej opcji powoduje wy¶wietlenie wszystkich stron
-manuala, które pasuj± do
-.B nazwy, 
-nie tylko pierwszej.
+Domy¶lnie man koñczy dzia³anie po wy¶wietleniu pierwszej znalezionej
+strony. Podanie tej opcji powoduje wy¶wietlenie wszystkich stron
+podrêcznika, które pasuj± do
+.BR nazwa , 
+a nie tylko pierwszej.
 .TP
 .B \-\^c
-Przformatuj ¼ród³ow± stronê manuala, nawet jezeli istnije aktualna
-storna cat. Mo¿e to byæ istotne, je¿eli strona cat jest sformatowana
+Przeformatowuje ¼ród³ow± stronê podrêcznika, nawet je¿eli istnieje aktualna
+strona cat. Mo¿e to byæ istotne, je¿eli strona cat jest sformatowana
 dla ekranu z inn± liczb± kolumn.
 .TP
 .B \-\^d
-Zamiast stron manuala wy¶wietl mnóstwo informacji o wykonywaniu
-zadania.
+Zamiast stron podrêcznika wy¶wietla mnóstwo informacji diagnostycznych.
 .TP
 .B \-\^D
-Wy¶wietl zarówno stronê manuala jak i informacjê o wykonaniu.
+Wy¶wietla zarówno stronê podrêcznika jak i informacje diagnostyczne.
 .TP
 .B \-\^f
 Równowa¿ne
-.B whatis.
+.BR whatis .
 .TP
 .B \-\^h
-Wy¶wietl krótk± informacjê o tym jak u¿ywaæ programu i zakoñcz pracê.
+Wy¶wietla krótk± informacjê pomocy i koñczy pracê.
 .TP
 .B \-\^k
-Równowa¿ny
-.B apropos.
+Równowa¿ne
+.BR apropos .
 .TP
 .B \-\^K
-Szukaj podanego ³añcucha znaków we *wszystkich* stronach manuala. Uwaga:
-mo¿e byæ bardzo powolne! Podanie sekscji pomaga przyspieszyæ szukanie.
-(Dla ogólnej orientacji: na mojej maszynie zajmuje to ok. minuty dla
-500 stron manuala.)
+Szuka podanego ³añcucha znaków we *wszystkich* stronach podrêcznika. Uwaga:
+bêdzie to prawdopodobnie bardzo powolne! Podanie rozdzia³ów pomaga
+przyspieszyæ szukanie. (Dla ogólnej orientacji: na mojej maszynie zajmuje to
+ok. minuty na ka¿de 500 stron podrêcznika.)
 .TP
 .B \-\^m " system"
-Podanie alternatywnego zestawu stron manuala, odpowiedniego dla
-podanego systemu.
+Podanie alternatywnego zestawu stron podrêcznika, odpowiedniego dla
+systemu o podanej nazwie.
 .TP
 .B \-\^p " ³añcuch_znaków"
-Podanie nazw preprocesorów, ktore zostan± uruchomione przed nroff i
-troff. Nie wszystkie intalacje maj± pe³ny zestaw preprocesorów.
-Niektóre z nich i litery u¿ywane do ich oznaczenia to:
+Podanie listy nazw preprocesorów, ktore zostan± uruchomione przed
+.B nroff
+i
+.BR troff .
+Nie wszystkie intalacje maj± pe³ny zestaw preprocesorów.
+Niektóre z nich oraz oznaczaj±ce je litery to:
 eqn (e), grap (g), pic (p), tbl (t), vgrind (v), refer (r).
 Podanie tej opcji powoduje zignorowanie zmiennej ¶rodowiskowej
-.B MANROFFSEQ
-.
+.BR MANROFFSEQ .
 .TP
 .B \-\^t
-U¿yj
-.B @troff@
-dla sformatowania strony manuala, a wynik wy¶lij do
+Sformatuj stronê podrêcznika za pomoc±
+.BR @troff@ ,
+a wynik wy¶lij na
 .B stdout
-(standardowego wyj¶cia).
-Wynik z
+(standardowe wyj¶cie).
+Mo¿e byæ konieczne przepuszczenie wyniku z
 .B @troff@
-powinien byæ na ogó³ przepuszczony przez jaki¶ filtr przed wydrukowaniem.
+przez jaki¶ filtr przed wydrukowaniem.
 .TP
 .B \-\^w \fRlub\fP \-\-path
-Nie wy¶wietlaj stron manuala, podaj tylko gdzie znajduj± siê pliki, które
-by³yby sformatowane i wyswietlone. Je¿eli nie s± podane argumenty:
-wy¶wietla (przed stdout) liste katalogow przeszukiwanych przez
+Nie wy¶wietlaj stron podrêcznika, podaj tylko gdzie znajduj± siê pliki, które
+by³yby sformatowane i wy¶wietlone. Je¿eli nie podano ¿adnych argumentów:
+wy¶wietla (na stdout) listê katalogów, w których
 .B man
-. Je¿eli
+poszukuje stron podrêcznika. Je¿eli
 .B manpath
-to link do man to "manpath" jest równowa¿ne "man --path".
+jest dowi±zaniem man to "manpath" jest równowa¿ne "man --path".
+.TP
+.B \-\^W
+Podobne do \-\^w, lecz podaje nazwy plików, ka¿d± w osobnej linii, bez
+dodatkowych informacji. Przydaje siê w poleceniach pow³oki, np.
+.ft CW
+.B "man -aW man | xargs ls -l"
+.ft
 
 .SH "STRONY CAT"
-Man bêdzie próbowal zapisaæ sformatowane strony manuala aby
-zaoszczedziæ czasu zu¿ywanego na formatowanie nastepnym razem,
-kiedy te strony bêd± potrzebne.
-Tradycyjnie sformatowane wersje stron z katalogu DIR/manX s±
-zapisywane do DIR/catX, ale inne w pliku @man_config_file@
-mog± byæ zapisane równie¿ inne zasady.
-Strony cat nie s± zapisywane je¿eli wymagany do tego katalog nie istnieje.
-.TP
-Mo¿liwe jest zmiana uprawnien man-a na suid man. Wtedy katalogi cat
-maj± w³a¶ciciela man i uprawnienia 0755 (zapisywalne tylko przez
-u¿ytkownika man), a pliki cat maj± w³a¶ciciela man i uprawnienia
-0644 lub 0444 (zapisywalne przez man lub przez nikogo). ¯aden zwyk³y
-u¿ytkownik nie mo¿e zmieniæ stron cat lub wpisaæ innych plików w te
-katalogi. Je¿eli man nie ma uprawnien suid to katalog cat powinien mieæ
-uprawnienia 0777, je¿eli wszyscy u¿ytkownicy powinni mieæ mo¿liwo¶æ
-pozostawienia tam stron cat.
-.TP
-Opcja -c wymusza przeformatowanie strony nawet je¿eli istnieje ju¿
-strona cat.
+Man bêdzie próbowa³ zachowaæ sformatowane strony podrêcznika aby
+skróciæ czas potrzebny na sformatowanie ich po raz kolejny,
+kiedy te strony bêd± znowu potrzebne.
+Tradycyjnie, sformatowane wersje stron podrêcznika z katalogu DIR/manX s±
+zachowywane w DIR/catX, ale mo¿na te¿ w pliku
+.BR @man_config_file@
+okre¶liæ inne zasady.
+Strony cat nie s± tworzone, je¿eli wymagany katalog nie istnieje.
+.PP
+Mo¿liwe jest uczynienie programu
+.B man
+suidowym na u¿ytkownika man. Wtedy, je¶li katalogi cat maj± w³a¶ciciela man
+i prawa dostêpu 0755 (zapisywalne tylko przez u¿ytkownika man), a pliki cat
+maj± w³a¶ciciela man i prawa dostêpu 0644 lub 0444 (zapisywalne przez
+u¿ytkownika man lub przez nikogo), to ¿aden zwyk³y u¿ytkownik nie mo¿e
+zmieniaæ stron cat lub umieszczaæ w katalogach cat innych plików. Je¿eli
+.B man
+nie jest suidowy to katalog cat powinien mieæ prawa dostêpu 0777,
+o ile wszyscy u¿ytkownicy powinni mieæ mo¿liwo¶æ pozostawiania tam stron cat.
+.PP
+Opcja
+.B \-c
+wymusza przeformatowanie strony nawet je¿eli istnieje ju¿ odpowiednia strona
+cat.
 
 
-.SH "ZMIENNE ¦RODOWISKOWE"
-.TP
-.B MANPATH
-Je¿eli
+.SH "¦CIE¯KA PRZESZUKIWANIA DLA STRON PODRÊCZNIKA"
+.B man
+stosuje wymy¶ln± metodê poszukiwania plików stron podrêcznika, opart± na
+opcjach wywo³ania, zmiennych ¶rodowiskowych, pliku konfiguracyjnym
+.B @man_config_file@
+oraz na pewnych wbudowanych konwencjach i heurystyce.
+.PP
+Najpierw, gdy argument
+.I nazwa
+polecenia
+.B man
+zawiera uko¶nik
+.RB ( / ),
+.B man
+zak³ada, ¿e jest on nazw± w³a¶ciwego pliku i nie prowadzi poszukiwania.
+.PP
+Ale zazwyczaj, gdy
+.I nazwa
+nie zawiera uko¶nika,
+.B man
+poszukuje pliku, który móg³by stanowiæ stronê podrêcznika o zadanym temacie,
+w ró¿nych katalogach.
+.PP
+Je¶li zostanie podana opcja
+.BI "-M " ¶cie¿ka\fR,
+to
+.I ¶cie¿ka
+jest rozdzielon± dwukropkami list± przeszukiwanych przez program
+.B man
+katalogów.
+.PP
+Gdy opcja
+.B -M
+nie zostanie podana, ale zostanie zdefiniowana zmienna ¶rodowiskowa
+.BR MANPATH ,
+warto¶æ tej zmiennej stanowi listê przeszukiwanych przez program
+.B man
+katalogów.
+.PP
+Je¶li ¶cie¿ka nie zostanie podana w sposób jawny ani za pomoc±
+.B -M
+ani poprzez
+.BR MANPATH ,
+.B man
+rozwija swoj± w³asn± ¶cie¿kê w oparciu o zawarto¶æ pliku konfiguracyjnego
+.BR @man_config_file@ .
+Instrukcje
 .B MANPATH
-jest zdefiniowana, warto¶æ jest u¿ywana jako ¶cie¿ka poszukiwania stron
-manuala.
+w pliku konfiguracyjnym okre¶laj± poszczególne katalogi w³±czane do ¶cie¿ki
+przeszukiwania.
+.PP
+Ponadto, instrukcje
+.B MANPATH_MAP
+dodaj± do ¶cie¿ki przeszukiwania katalogi zale¿ne od ¶cie¿ki przeszukiwania
+dla poleceñ (tzn. zawartej w zmiennej ¶rodowiskowej
+.B PATH
+).
+Instrukcja
+.B MANPATH_MAP
+dodaje do ¶cie¿ki przeszukiwania dla stron podrêcznika po jednym katalogu dla
+ka¿dego katalogu zawartego w ¶cie¿ce przeszukiwania dla poleceñ.
+.B man
+przegl±da zmienn±
+.B PATH
+i dodaje odpowiednie katalogi do ¶cie¿ki przeszukiwania dla stron podrêcznika.
+Zatem przy w³a¶ciwym u¿yciu
+.BR MANPATH_MAP ,
+po wydaniu polecenia
+.BR "man xyz" ,
+otrzyma siê stronê podrêcznika dla programu, który zosta³by uruchomiony
+poprzez wydanie polecenia
+.BR xyz .
+.PP
+Dodatkowo, dla ka¿dego katalogu w ¶cie¿ce przeszukiwania dla poleceñ
+(bêdziemy go nazywaæ "katalogiem poleceñ"), dla którego
+.I nie
+ma instrukcji
+.BR MANPATH_MAP ,
+.B man
+automatycznie poszukuje "bliskiego" katalogu stron podrêcznika jako
+podkatalogu w³a¶ciwego katalogu poleceñ lub w katalogu nadrzêdnym dla katalogu
+poleceñ.
+.PP
+Mo¿na wy³±czyæ automatyczne "bliskie" przeszukiwania do³±czaj±c instrukcjê
+.B NOAUTOPATH
+do
+.BR @man_config_file@ .
+.PP
+Jak opisano powy¿ej, w ka¿dym z katalogów ¶cie¿ki przeszukiwania
+.B man
+poszukuje pliku o nazwie
+.IB tytu³ . rozdzia³\fR,
+z opcjonalnym przyrostkiem dla numeru rozdzia³u i mo¿liwym przyrostkiem
+kompresji. Je¶li nie znajdzie takiego pliku, bêdzie szuka³ w podkatalogach 
+o nazwach
+.BI man N
+i
+.BI cat N\fR,
+gdzie
+.I N
+jest numerem rozdzia³u podrêcznika.
+Je¶li plik znajduje siê w podkatalogu
+.BIR cat N ,
+.B man
+zak³ada, ¿e jest to sformatowany plik strony podrêcznika (cat page). 
+W przeciwnym przypadku,
+.B man
+zak³ada, ¿e jest ona niesformatowana. W obu przypadkach, je¶li nazwa pliku
+zawiera znany przyrostek kompresji (jak
+.BR .gz ),
+.B man
+zak³ada, ¿e jest ona spakowania gzipem.
+.PP
+Aby zobaczyæ, gdzie (lub czy)
+.B man
+znajdzie stronê podrêcznika o okre¶lonym tytule, nale¿y pos³u¿yc siê opcj±
+.BR "--path " ( -w ).
+
+.SH "¦RODOWISKO"
+.TP
+.B MANPL
+Je¶li zmienna
+.B MANPL
+jest zdefinowana, to jej warto¶æ jest u¿ywana do okre¶lenia d³ugo¶ci strony.
+W przeciwnym przypadku, ca³a strona podrêcznika bêdzie stanowiæ jedn± (d³ug±)
+stronê.
 .TP
 .B MANROFFSEQ
-Je¿eli
+Je¶li zmienna
 .B MANROFFSEQ
-jest zdefinowana, warto¶æ jest u¿ywana do zdefinowania zestawu
-preprocesorów uruchamianych przed nroff i troff. Domy¶lnie strony
-s± przepuszczane przez procesor tablic przed u¿yciem nroff.
+jest zdefinowana, to jej warto¶æ jest u¿ywana do zdefinowania zestawu
+preprocesorów uruchamianych przed
+.B nroff
+i
+.BR troff .
+Domy¶lnie strony s± przepuszczane przez preprocesor tbl przed u¿yciem nroff.
 .TP
 .B MANSECT
-Je¿eli
+Je¶li zmienna
 .B MANSECT
-jest zdefiniowana, warto¶æ jest u¿ywana do podania jakie sekcje manuala
-powinny zostac przeszukane.
+jest zdefiniowana, to jej warto¶æ jest u¿ywana do okre¶lenia jakie rozdzia³y
+podrêcznika powinny zostaæ przeszukane.
 .TP
-.B PAGER
-Je¿eli
-.B PAGER
-jest zdefiniowana, warto¶æ jest u¿ywana jako nazwa programu do
-wy¶wietlania strony manuala. Domy¶lnie uzywany jest
-.B @pager@
-.
+.B MANWIDTH
+Je¶li zmienna
+.B MANWIDTH
+jest zdefiniowana, to jej warto¶æ jest u¿ywana jako szeroko¶æ wy¶wietlanych
+stron podrêcznika man. W przeciwnym wypadku, strony bêd± wy¶wietlane na ca³ej
+szeroko¶ci ekranu.
+.TP
+.B MANPAGER
+Je¶li zmienna
+.B MANPAGER
+jest zdefiniowana, to jej warto¶æ jest u¿ywana jako nazwa programu do
+wy¶wietlania strony podrêcznika man. Je¶li nie jest zdefiniowana, to u¿ywana
+jest zmienna
+.BR PAGER .
+Je¿eli ta równie¿ nie ma nadanej warto¶ci, to u¿ywany jest
+.BR @pager@ .
 .TP
 .B LANG
-Je¿eli
+Je¶li zmienna
 .B LANG
-jest zdefiniowana, warto¶æ jest nazw± podkatalogu, gdzie man szuka
-stron. Zatem komenda `LANG=dk man 1 foo' (w sh lub bash) spowoduje, ¿e
-man bêdzie szukal strony w .../dk/man1/foo.1, a nastêpnie, je¿eli nie
-bêdzie móg³ znale¼æ takiego pliku, w .../man1/foo.1, gdzie ... jest
-katalogiem ze ¶cie¿ki przeszukiwania.
+jest zdefiniowana, to jej warto¶æ okre¶la nazwê podkatalogu, w którym man
+najpierw bêdzie szukaæ stron podrêcznika. Zatem polecenie `LANG=pl man 1 co¶'
+(w sh lub bash) spowoduje, ¿e man bêdzie szukaæ strony "co¶"
+w .../pl/man1/co¶.1, a nastêpnie, je¿eli takiego pliku nie znajdzie,
+w .../man1/co¶.1, gdzie ... jest katalogiem ze ¶cie¿ki przeszukiwania.
 .TP
 .B "NLSPATH, LC_MESSAGES, LANG"
 Zmienne ¶rodowiskowe
@@ -210,27 +381,47 @@
 i
 .B LC_MESSAGES
 (lub
-.B LANG,
+.BR LANG ,
 je¿eli ta druga nie istnieje)
 steruj± przeszukiwaniem katalogów z komunikatami.
 (Komunikaty angielskie s± wkompilowane, wiêc dla angielskiego taki
 katalog nie jest potrzebny.)
-Zauwa¿, ze programy takie jak col(1), wo³ane przez man, równie¿ u¿ywaj±
-np. LC_TYPE.
+Zauwa¿, ¿e programy takie jak
+.BR col(1) ,
+wo³ane przez man, równie¿ u¿ywaj± np. LC_TYPE.
 .TP
 .B PATH
 .B PATH
-jest u¿ywana do zbudowania domy¶lnej ¶ciezki szukania stron manuala.
+pomaga okre¶liæ ¶cie¿kê przeszukiwania dla plików stron podrêcznika.
+Zobacz
+.BR "¦CIE¯KA PRZESZUKIWANIA DLA STRON PODRÊCZNIKA" .
 .TP
 .B SYSTEM
 .B SYSTEM
-jest u¿ywana dla podania domy¶lnej alternatywnej nazwy systemu (do u¿ywania
+jest u¿ywana do pobierania domy¶lnej alternatywnej nazwy systemu (do u¿ywania
 razem z opcj±
-.B \-m
-). 
-.SH "ZOBACZ TE¯"
-apropos(1), whatis(1), less(1), groff(1).
+.BR \-m ). 
+.SH "ZOBACZ TAK¯E"
+apropos(1), whatis(1), less(1), groff(1), man.conf(5).
 .SH B£ÊDY
 Opcja
 .B \-t
 dzia³a tylko je¿eli jest zainstalowany jaki¶ program dzia³aj±cy jak troff.
+.br
+Je¿eli zamiast my¶lników pojawi± siê migaj±ce \e255 lub <AD>, to nale¿y
+ustawiæ w ¶rodowisku u¿ytkownika `LESSCHARSET=latin1'.
+.SH ZAKOÑCZENIE
+Je¶li do pliku
+.IR .emacs
+u¿ytkownika dodana zostanie linia
+
+  (global-set-key [(f1)] (lambda () (interactive) (manual-entry (current-word))))
+
+to naci¶niêcie F1 spowoduje wy¶wietlenie strony podrêcznika dla funkcji
+bibliotecznej, na której ustawiony jest kursor.
+.LP
+Aby uzyskaæ czysto tekstow± wersjê strony podrêcznika, bez cofniêæ
+i podkre¶leñ, nale¿y wydaæ polecenie
+
+  # man co¶ | col -b > co¶.mantxt
+
