diff -Nur man-1.5k.orig/src/makewhatis.sh man-1.5k/src/makewhatis.sh
--- man-1.5k.orig/src/makewhatis.sh	Mon Mar 19 17:15:24 2001
+++ man-1.5k/src/makewhatis.sh	Sun Oct 14 00:41:32 2001
@@ -36,8 +36,8 @@
 # In case both /usr/man and /usr/share/manexist, the former is local
 # and should be first.
 # It is a bug to add /var/cache/man to DEFCATPATH.
-DEFMANPATH=/usr/man:/usr/share/man
-DEFCATPATH=/usr/man/preformat:/usr/man:/usr/share/man/preformat:/usr/share/man
+DEFMANPATH=/usr/share/man
+DEFCATPATH=/usr/share/man/preformat:/usr/share/man
 
 # In case /usr is read-only, make /usr/man/whatis (etc) a symlink to
 # something like /var/cache/man/whatis.
