--- man-1.5l/src/makewhatis.sh.orig	Mon Mar 10 21:14:51 2003
+++ man-1.5l/src/makewhatis.sh	Tue Mar 11 21:50:50 2003
@@ -41,7 +41,7 @@
 # and should be first.
 # It is a bug to add /var/cache/man to DEFCATPATH.
 dm=
-for d in /usr/man /usr/share/man
+for d in /usr/share/man
 do
     if [ -d $d ]; then
 	if [ x$dm = x ]; then dm=$d; else dm=$dm:$d; fi
@@ -49,7 +49,7 @@
 done
 DEFMANPATH=$dm
 dc=
-for d in /usr/man/preformat /usr/man /usr/share/man/preformat /usr/share/man
+for d in /usr/share/man/preformat /usr/share/man
 do
     if [ -d $d ]; then
 	if [ x$dc = x ]; then dc=$d; else dc=$dc:$d; fi
@@ -391,13 +391,7 @@
 
        cd $here
 
-       # kludge for Slackware's /usr/man/preformat
-       if [ $mandir = /usr/man/preformat ]
-       then
-	 mandir1=/usr/man
-       else
-	 mandir1=$mandir
-       fi
+       mandir1=$mandir
 
        if [ -f ${mandir1}/whatis ]
        then
