--- man-1.5p/src/makewhatis.sh	2005-01-08 18:04:00.000000000 +0000
+++ man-1.5p.new/src/makewhatis.sh	2005-03-07 21:44:16.818469600 +0000
@@ -156,7 +156,7 @@
       eval path="\$$pages"path
       for mandir in $path
       do
-	 cp /dev/null $mandir/whatis
+	 cp /dev/null /var/cache/man/whatis
       done
    done
 fi
@@ -170,7 +170,7 @@
      if [ x$verbose != x ]; then
 	echo "about to enter $mandir" > /dev/stderr
      fi
-     if [ -s ${mandir}/whatis -a $pages = man -a x$update = x ]; then
+     if [ -s /var/cache/man/whatis -a $pages = man -a x$update = x ]; then
 	if [ x$verbose != x ]; then
 	   echo skipping $mandir - we did it already > /dev/stderr
 	fi
@@ -401,13 +401,13 @@
 	 mandir1=$mandir
        fi
 
-       if [ -f ${mandir1}/whatis ]
+       if [ -f /var/cache/man/whatis ]
        then
-	 cat ${mandir1}/whatis >> $TMPFILE
+	 cat /var/cache/man/whatis >> $TMPFILE
        fi
        tr -s '\n' < $TMPFILE | sort -u > ${mandir1}/whatis
 
-       chmod 644 ${mandir1}/whatis
+       chmod 644 /var/cache/man/whatis
        rm $TMPFILE
      fi
    done
