--- man-1.5k/configure.orig	Sun Oct 14 00:49:58 2001
+++ man-1.5k/configure	Sun Oct 14 00:52:35 2001
@@ -362,20 +362,20 @@
   then
     if test $Fnroff = "missing"
     then
-      nroff="nroff -mandoc"
+      nroff="nroff -msafer -Tlatin1 -mandoc"
     else
-      nroff="$Fnroff -mandoc"
+      nroff="$Fnroff -msafer -Tlatin1 -mandoc"
     fi
-    troff="troff -mandoc"
+    troff="troff -msafer -mandoc"
     echo "Warning: could not find groff"
   else
     if test $Fnroff = "missing"
     then
-      nroff="$Fgroff -Tlatin1 -mandoc"
+      nroff="$Fgroff -S -Tlatin1 -mandoc"
     else
-      nroff="$Fnroff -mandoc"
+      nroff="$Fnroff -msafer -Tlatin1 -mandoc"
     fi
-    troff="$Fgroff -Tps -mandoc"
+    troff="$Fgroff -S -Tps -mandoc"
     jnroff="$Fgroff -Tnippon -mandocj"
   fi
   eqn="$Fgeqn -Tps"
