diff -Nru man-1.5g/configure man-1.5g.new/configure
--- man-1.5g/configure	Sat Sep 25 17:06:39 1999
+++ man-1.5g.new/configure	Sat Sep 25 17:07:58 1999
@@ -528,8 +528,8 @@
 mandir="usr/man"
 confdir="etc"
 if [ x"$fhs" = x ]; then
-  mandir="share/man"
-  confdir="share/misc"
+  mandir="usr/share/man"
+  confdir="etc"
 fi
 
 if [ x$default = x ]; then
@@ -681,7 +681,7 @@
 
 if [ "x$languages" != x -a "x$languages" != xen ]
 then
-  DEF_NLSPATH="/usr/lib/locale/%N/%L:/usr/share/locale/%L/%N"
+  DEF_NLSPATH="/usr/share/locale/%L/%N"
   deflocalepath=`echo $NLSPATH:$DEF_NLSPATH | sed 's/:/ /g'`
   for i in $deflocalepath; do
     if [ x$deflocale = x ]; then
@@ -697,7 +697,7 @@
     fi
   done
   if [ x$deflocale = x ]; then
-    deflocale="/usr/lib/locale/%N/%L"
+    deflocale="/usr/share/locale/%N/%L"
   fi
   if [ x$default = x ]; then
     echo ""
diff -Nru man-1.5g/catopen/catopen.c man-1.5g.new/catopen/catopen.c
--- man-1.5g/catopen/catopen.c	Wed Apr  7 03:37:29 1999
+++ man-1.5g.new/catopen/catopen.c	Sat Sep 25 17:19:44 1999
@@ -9,7 +9,7 @@
 extern char *index (const char *, int);         /* not always in <string.h> */
 extern char *my_malloc(int);	/* in util.c */
 
-#define DEFAULT_NLSPATH "/usr/lib/locale/%N/%L"
+#define DEFAULT_NLSPATH "/usr/share/locale/%N/%L"
 
 static nl_catd my_catopenpath(char *name, char *path);
 
