diff -Nur man-1.5k.orig/catopen/catopen.c man-1.5k/catopen/catopen.c
--- man-1.5k.orig/catopen/catopen.c	Mon Mar 19 21:37:38 2001
+++ man-1.5k/catopen/catopen.c	Sun Oct 14 00:48:37 2001
@@ -10,7 +10,7 @@
 extern char *my_malloc(int);	/* in util.c */
 
 #ifndef DEFAULT_NLSPATH
-#define DEFAULT_NLSPATH "/usr/lib/locale/%N/%L"
+#define DEFAULT_NLSPATH "/usr/share/locale/%N/%L"
 #endif
 
 static nl_catd my_catopenpath(char *name, char *path);
diff -Nur man-1.5k.orig/man2html/Makefile.in man-1.5k/man2html/Makefile.in
--- man-1.5k.orig/man2html/Makefile.in	Sun Oct 14 11:17:22 2001
+++ man-1.5k/man2html/Makefile.in	Mon Oct 15 22:01:51 2001
@@ -28,13 +28,13 @@
 	mkdir -p $(DESTDIR)/$(httpdir)/cgi-aux/man
 	install -m 755 scripts/cgi-bin/man/* $(DESTDIR)/$(httpdir)/cgi-bin/man
 	install -m 644 scripts/cgi-aux/man/* $(DESTDIR)/$(httpdir)/cgi-aux/man
-	install -d -m 775 $(DESTDIR)/$(vardir)/man2html
+	install -d -m 775 $(DESTDIR)/$(vardir)/cache/man2html
 # (aux was renamed to cgi-aux since aux causes problems under DOS)
 
 # If you have installed glimpse, and have compressed man pages,
 # then perhaps you also want these filters.
 install-glimpse-stuff:
-	install -m 644 glimpse_filters $(DESTDIR)/$(vardir)/man2html/.glimpse_filters
+	install -m 644 glimpse_filters $(DESTDIR)/$(vardir)/cache/man2html/.glimpse_filters
 
 # In order not to have to type a long command like
 #   netscape http://localhost/cgi-bin/man/man2html?section+topic
diff -Nur man-1.5k.orig/man2html/scripts/cgi-bin/man/mansearch man-1.5k/man2html/scripts/cgi-bin/man/mansearch
--- man-1.5k.orig/man2html/scripts/cgi-bin/man/mansearch	Mon Mar 19 17:14:06 2001
+++ man-1.5k/man2html/scripts/cgi-bin/man/mansearch	Mon Oct 15 22:00:21 2001
@@ -62,7 +62,7 @@
   sedcmd = ENVIRON["SED"];
   truncate_at = 11;		# Single page display match limit.
 
-  glimpse_cmd = "glimpse -z -H /var/man2html -y -W -i "
+  glimpse_cmd = "glimpse -z -H /var/cache/man2html -y -W -i "
 
   for (i = 1; i < ARGC; i++) {
     string = string " " ARGV[i];
diff -Nur man-1.5k.orig/man2html/scripts/cgi-bin/man/mansec man-1.5k/man2html/scripts/cgi-bin/man/mansec
--- man-1.5k.orig/man2html/scripts/cgi-bin/man/mansec	Sun Mar  1 13:25:40 1998
+++ man-1.5k/man2html/scripts/cgi-bin/man/mansec	Mon Oct 15 22:00:11 2001
@@ -40,7 +40,7 @@
     cache_suffix = "h";
   }
 
-  cache_dir  = "/var/man2html";
+  cache_dir  = "/var/cache/man2html";
   cache_file = "manindex" cache_suffix "-" section ".html";
   cache = cache_dir "/" cache_file;
   cache_tmp = cache "_" pid;
diff -Nur man-1.5k.orig/man2html/scripts/cgi-bin/man/manwhatis man-1.5k/man2html/scripts/cgi-bin/man/manwhatis
--- man-1.5k.orig/man2html/scripts/cgi-bin/man/manwhatis	Sat Sep 12 18:14:05 1998
+++ man-1.5k/man2html/scripts/cgi-bin/man/manwhatis	Mon Oct 15 22:00:02 2001
@@ -39,7 +39,7 @@
     cache_suffix = "h";
   }
 
-  cache_dir  = "/var/man2html";
+  cache_dir  = "/var/cache/man2html";
   cache_file = "whatis" cache_suffix "-" section ".html";
   cache = cache_dir "/" cache_file;
 
