--- man-1.5p/man2html/Makefile.in	2005-01-08 00:49:37.000000000 +0000
+++ man-1.5p.new/man2html/Makefile.in	2005-03-07 21:33:04.408691424 +0000
@@ -31,7 +31,7 @@
 	mkdir -p $(httpdir)/cgi-aux/man
 	install -m 755 scripts/cgi-bin/man/* $(httpdir)/cgi-bin/man
 	install -m 644 scripts/cgi-aux/man/* $(httpdir)/cgi-aux/man
-	install -d -o $(cgiowner) -g $(cgigroup) -m 775 $(vardir)/man2html
+	install -d -m 775 $(vardir)/man2html
 # (aux was renamed to cgi-aux since aux causes problems under DOS)
 
 # If you have installed glimpse, and have compressed man pages,
