diff -Naur man-1.5h1/src/makewhatis.sh man-1.5h1-p/src/makewhatis.sh
--- man-1.5h1/src/makewhatis.sh	Tue Jun 29 04:20:59 1999
+++ man-1.5h1-p/src/makewhatis.sh	Fri Jul 14 17:39:43 2000
@@ -36,8 +36,8 @@
 DEFMANPATH=/usr/man
 DEFCATPATH=/usr/man/preformat:/usr/man
 
-# AWK=/usr/bin/gawk
-AWK=%gawk%
+AWK=/bin/awk
+# AWK=%gawk%
 
 # Find a place for our temporary files. If security is not a concern, use
 #	TMPFILE=/tmp/whatis$$; TMPFILEDIR=none
