--- man-1.5k/configure.orig	Wed Sep 16 22:57:54 1998
+++ man-1.5k/configure	Wed Sep 16 23:13:32 1998
@@ -144,7 +144,7 @@
 # All documentation assumes "man.conf" - in case you invent some other name
 # (why?), also man pages and other docs need to be patched.
 prefix="/usr"
-confprefix="/usr"
+confprefix=""
 bindir="bin"
 sbindir="sbin"
 mandir="man"
@@ -159,7 +159,7 @@
 else
   confprefix=""
 fi
-conffilename="man.conf"
+conffilename="man.config"
 
 if test "$RANDOM" = "$RANDOM"; then
   # Plain old Bourne shell.
