diff -Nur man-1.5i2.orig/src/man.c man-1.5i2/src/man.c
--- man-1.5i2.orig/src/man.c	Wed Jun  6 00:01:47 2001
+++ man-1.5i2/src/man.c	Mon Oct 15 22:19:36 2001
@@ -1106,6 +1106,21 @@
      extern int optind;
 #endif
 
+     char *s;
+#define CHECK(p, l) s=getenv(p); if(s && (strlen(s)>l)) { fprintf(stderr, "ERROR: Environment variable %s too long!\n", p); exit(1); }
+
+     CHECK("LANG", 32);
+     CHECK("MANPAGER", 128);
+     CHECK("PAGER", 128);
+     CHECK("SYSTEM", 64);
+     CHECK("MANROFFSEQ", 128);
+     CHECK("MANSECT", 128);
+     CHECK("MANPL", 128);
+     CHECK("MAN_HP_DIREXT", 128);
+     CHECK("LANGUAGE", 128);
+     CHECK("LC_MESSAGES", 128);
+     CHECK("LC_ALL", 128);
+
 #ifndef __FreeBSD__ 
      /* Slaven Rezif: FreeBSD-2.2-SNAP does not recognize LC_MESSAGES. */
      setlocale(LC_MESSAGES, "");
