diff -Nur man-1.5k.orig/catopen/catopen.c man-1.5k/catopen/catopen.c
--- man-1.5k.orig/catopen/catopen.c	Sun Oct 14 11:05:49 2001
+++ man-1.5k/catopen/catopen.c	Sun Oct 14 11:06:00 2001
@@ -27,14 +27,6 @@
     /* The quotes below are from X/Open, XPG 1987, Vol. 3. */
 
     /*
-     * "In catopen(), the oflag argument is reserved for future use
-     * and should be set to 0."
-     */
-
-    if (oflag)
-      return fd;		/* only treat the known case */
-
-    /*
      * "If `name' contains a `/', then `name' specifies a pathname"
      */
     if (index(name, '/')) {
@@ -87,8 +79,12 @@
 
   namesz = strlen(name);
 
-  lang = getenv("LANG");
+  lang = getenv("LC_ALL");
+  if (!lang)
+    lang = getenv("LC_MESSAGES");
   if (!lang)
+    lang = getenv("LANG");
+  if (!lang || !strcmp(lang,"C") || !strcmp(lang,"POSIX"))
     lang = "";
   langsz = strlen(lang);
 
--- man-1.5p/src/gripes.c.orig	2005-01-08 09:01:21.000000000 +0100
+++ man-1.5p/src/gripes.c	2005-03-09 22:43:31.254314432 +0100
@@ -49,12 +49,12 @@
 	     */
 	    char *s, *lg;
 	    s = getenv("NLSPATH");
-	    lg = getenv("LANG");
+	    lg = getenv("LC_ALL");
 	    if (!lg)
 		    lg = getenv("LC_MESSAGES");
 	    if (!lg)
-		    lg = getenv("LC_ALL");
-	    if (lg && strncmp(lg, "en", 2) && strcmp(lg, "C")) {
+		    lg = getenv("LANG");
+	    if (lg && strncmp(lg, "en", 2) && strcmp(lg, "C") && strcmp(lg, "POSIX")) {
 		fprintf(stderr,
 		  "Cannot open the message catalog \"%s\" for locale \"%s\"\n"
 		  "(NLSPATH=\"%s\")\n\n",
diff -Nur man-1.5k.orig/src/man.c man-1.5k/src/man.c
--- man-1.5k.orig/src/man.c	Sun Oct 14 11:05:49 2001
+++ man-1.5k/src/man.c	Sun Oct 14 11:06:00 2001
@@ -1217,11 +1217,7 @@
      CHECK("LC_MESSAGES", 128);
      CHECK("LC_ALL", 128);
 
-#ifndef __FreeBSD__ 
-     /* Slaven Rezif: FreeBSD-2.2-SNAP does not recognize LC_MESSAGES. */
-     setlocale(LC_CTYPE, "");	/* used anywhere? maybe only isdigit()? */
-     setlocale(LC_MESSAGES, "");
-#endif
+     setlocale(LC_ALL, "");
 
      /* No doubt we'll need some generic language code here later.
 	For the moment only Japanese support. */
--- man-1.5l/src/manpath.c.orig	Mon Mar 10 17:46:24 2003
+++ man-1.5l/src/manpath.c	Tue Mar 11 22:00:24 2003
@@ -281,14 +281,12 @@
 		/* We cannot use "lang = setlocale(LC_MESSAGES, NULL)" or so:
 		   the return value of setlocale is an opaque string. */
 		/* POSIX prescribes the order: LC_ALL, LC_MESSAGES, LANG */
-		if((lang = getenv("LC_ALL")) != NULL)
-			split2(dir, lang, add_to_mandirlist_x, perrs);
-		if((lang = getenv("LC_MESSAGES")) != NULL)
-			split2(dir, lang, add_to_mandirlist_x, perrs);
-		if((lang = getenv("LANG")) != NULL)
-			split2(dir, lang, add_to_mandirlist_x, perrs);
-		if((lang = getenv("LANGUAGE")) != NULL)
-			split2(dir, lang, add_to_mandirlist_x, perrs);
+		lang = getenv("LC_ALL");
+		if(!lang) lang=getenv("LC_MESSAGES");
+		if(!lang) lang=getenv("LANG");
+		if(!lang) lang=getenv("LANGUAGE");
+		if(lang != NULL)
+  			split2(dir, lang, add_to_mandirlist_x, perrs);
 		add_to_mandirlist_x(dir, 0, perrs);
 	}
 }
