--- man-1.5l/configure.orig	Tue Mar 11 21:57:43 2003
+++ man-1.5l/configure	Tue Mar 11 21:58:03 2003
@@ -26,7 +26,7 @@
 # (Indeed, -r may cause the terminal to get into funny states.
 # Very inconvenient. For viewing pages in strange locales, set LC_*.)
 #
-DEFAULTLESSOPT="-is"
+DEFAULTLESSOPT="-isr"
 #
 # Note that not creating any cat directories (/var/cache/man or so)
 # and not making man suid or sgid is recommended.
