diff -ur man-1.5p/src/makewhatis.sh man-1.5p.new/src/makewhatis.sh
--- man-1.5p/src/makewhatis.sh	2005-01-08 18:04:00.000000000 +0000
+++ man-1.5p.new/src/makewhatis.sh	2005-03-07 21:29:44.671056160 +0000
@@ -41,7 +41,7 @@
 # and should be first.
 # It is a bug to add /var/cache/man to DEFCATPATH.
 dm=
-for d in /usr/man /usr/share/man /usr/X11R6/man /usr/local/man
+for d in /usr/share/man /usr/X11R6/man /usr/local/man
 do
     if [ -d $d ]; then
 	if [ x$dm = x ]; then dm=$d; else dm=$dm:$d; fi
@@ -49,7 +49,7 @@
 done
 DEFMANPATH=$dm
 dc=
-for d in /usr/man/preformat /usr/man /usr/share/man/preformat /usr/share/man
+for d in /usr/share/man/preformat /usr/share/man
 do
     if [ -d $d ]; then
 	if [ x$dc = x ]; then dc=$d; else dc=$dc:$d; fi

--- man-1.6f/src/makewhatis.sh~	2008-01-28 10:29:04.000000000 +0100
+++ man-1.6f/src/makewhatis.sh	2008-01-28 10:30:27.277670692 +0100
@@ -185,13 +185,7 @@
 	echo "about to enter $mandir" > /dev/stderr
      fi
 
-     # kludge for Slackware's /usr/man/preformat
-     if [ $mandir = /usr/man/preformat ]
-     then
-	mandir1=/usr/man
-     else
-	mandir1=$mandir
-     fi
+     mandir1=$mandir
 
      # if $mandir is on a readonly partition, and the whatis file
      # is not a symlink, then let's skip trying to update it
