--- man-1.5l/src/gripes.c.orig	Thu Jul 18 02:17:23 2002
+++ man-1.5l/src/gripes.c	Sat Jun  7 23:53:00 2003
@@ -26,6 +26,7 @@
 
 #else /* NONLS */
 
+#include <string.h>
 #include <nl_types.h>
 #include "../catopen/catopen.c"
 
@@ -66,6 +67,8 @@
 static char *
 getmsg (int n) {
     char *s;
+    unsigned int i = 0;
+    unsigned short fmt_n = 0;
 
     catinit ();
     if (catfd == (nl_catd) -1 || !*(s = catgets(catfd, 1, n, ""))) {
@@ -76,6 +79,19 @@
 	  exit (1);
         }
     }
+    /* routine to filter format string abuse.  will */
+    /* only allow %d, %s, and %o through.  no more  */
+    /* than two formats needed for any response.    */
+    for (i = 0; s[i] != 0x0; i++){
+        if (s[i] == '%' && s[i+1]){
+            if (strchr("dso", s[i+1])) /* %d,%s,%o. */
+                fmt_n++;
+            else
+                fmt_n=3; /* anything else = <limit. */
+        }
+        if (fmt_n > 2) /* failed, default reply. */
+            s = msg[n];
+    }
     return s;
 }
 
